processing/src/main/java/org/apache/druid/math/expr/ConstantExpr.java

```diff
@@ -157,9 +157,12 @@ public int hashCode()
 
 class LongExpr extends ConstantExpr<Long>
 {
+  final ExprEval expr;
```

Comment:
nit: private?

Reply:
I've added `private` - the other fields `value` and `outputType` are also package accessible